src/components/BooksQuery.tsx

```diff
@@ -53,13 +50,20 @@ const BooksQuery = () => {
         <button onClick={submit}>FIND</button>
       </div>
       <div className="books">
-        {list.map((book: book) => (
+        {list.books.map((book) => (
```

Comment:
`list.books` this naming seems awkward. Please use better names for your variables. For example, this `list` object seems unnecessary. you can just use a `books` array without putting it in an object